spring-cloud-dataflow-module-deployer-spi/src/main/java/org/springframework/cloud/dataflow/module/deployer/ModuleDeployer.java

```diff
@@ -38,6 +39,8 @@
 
 	public static final String JMX_DEFAULT_DOMAIN_KEY  = "spring.jmx.default-domain";
 
+	public static final String MODULE_DEPLOYMENT_GROUP_ID = "spring_cloud_dataflow.module_deployment_group_id";
```

Comment:
I wonder if "group-deployment-id" would be sufficient?... not only removing "module" as a bit redundant but reversing the other 2 terms as it's really an ID for a *group* having been *deployed*.